src/Raven.Client/Documents/Session/Tokens/GroupByKeyToken.cs

```diff
@@ -27,7 +27,7 @@ public override void WriteTo(StringBuilder writer)
 
             writer
                 .Append(" as ")
-                .Append(_projectedName);
+                .Append($"\'{_projectedName}\'");
```

Comment:
It should be:
```
writer
    .Append(" as '")
    .Append(_projectedName)
    .Append("'");   
```

You will avoid string allocation.

Reply:
Done